Externals.cfg

```diff
@@ -1,5 +1,5 @@
 [cam]
-branch = noresm2_3_develop
+branch = noresm2_3_chem
```

Comment:
I don't see a `noresm2_3_chem` branch in CAM. I see there is a PR to `noresm2_3_develop` from fullchem (https://github.com/NorESMhub/CAM/pull/149). I'm guessing this should point to `noresm2_3_develop`, but after CAM has been updated?

Reply:
Right. The idea is to allow testing at the NorESM level while the [OSLO_AERO](https://github.com/NorESMhub/OSLO_AERO/pull/29) and [CAM](https://github.com/NorESMhub/CAM/pull/149) PRs are in progress but for that to work  @DirkOlivie should change the `repo_url`:
```
repo_url = https://github.com/DirkOlivie/CAM
```
Once the CAM PR is merged, this file will revert and only the config_compsets.xml update will be in this PR.

Reply:
It is not clear to me how I should change Externals.cfg :  setting branch to noresm2_3_develop and repo_url to https://github.com/DirkOlivie/CAM might not work I think as in my fork noresm2_3_develop does not exist.

I would think that setting branch to noresm2_3_develop would be sufficient.  Am I missing something?

Should I make a complete new pull-request, or can I adapt it on github?

Reply:
Sorry, I was not being clear:
Now:
```
branch = noresm2_3_chem
repo_url = https://github.com/DirkOlivie/CAM
```
After [CAM PR](https://github.com/NorESMhub/CAM/pull/149) merge:
```
branch = noresm2_3_develop
repo_url = https://github.com/NorESMhub/CAM
```
Does this help?

Reply:
@DirkOlivie - If you want to point to `noresm2_3_chem` in `https://github.com/DirkOlivie/CAM`, you just need to change the `repo_url` path. You can do this by making a new commit in the current PR branch `[DirkOlivie:PR_fullchem](https://github.com/DirkOlivie/NorESM/tree/PR_fullchem), this PR will update automatically when you add to that branch.

Reply:
Once you have completed the testing and updated CAM and OSLO_AERO, I guess you can make a new PR changing the `Externals.cfg` back to the current form.

Reply:
I have now corrected the repo_url for CAM in Externals.cfg.  It points to https://github.com/DirkOlivie/CAM.  The branch is kept on noresm2_3_chem.